Sources/CyanImage/Color.swift

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
Shall we use `Image+Color.swift` as the file name to imply that this is an extension?